javascript/chronometer.js

```diff
@@ -1,36 +1,52 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
-  }
+    this.currentTime = 0;
+    this.intervalId = null;
+  };
 
   start(callback) {
     // ... your code goes here
-  }
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1
+      }, 1000
+    )
+  };
```

Comment:
inside set interval you must be checking if callback function is passed, and if it is-> you want to invoke it. 
You'll need this for next iterations :) 

```
  start(callback) {
    this.intervalId = setInterval(() => {
      this.currentTime++;
      if (callback) {
        callback();
      }
    }, 1000);
```